lib/widgets/Widgets.simba

```diff
@@ -135,12 +135,12 @@ end;
 
 function TReflectWidget.GetScrollX: Integer;
 begin
-  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_ScrollX);
+  Result := Reflect.Smart.GetFieldInt(Self.Reference, Widget_RelativeX);
```

Comment:
I see what you mean. You want the function renamed as well? I renamed the hook to Widget_RelativeX which is actually correct. I just didn't rename the function. Yeah I'll update it to take both into account.